app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt

```diff
@@ -0,0 +1,26 @@
+package io.mochahub.powermeter.models
+
+data class WorkoutSet (
+    val weight: Double,
+    val reps: Int = 0
+) {
+    operator fun plus(increment: Int): WorkoutSet {
+        return this.copy(reps = if (reps + increment <= 100) reps + increment else reps)
+    }
```

Comment:
What is this function doing?

Reply:
implements the `+` operator for a set so that I can do: `set = set + 2`